### ISSUE TITLE
The option always appear

### ISSUE THREAD

**Issue by fajarlahhh** [select2/select2#5504]
the option always appear and always focus to select2 element, even if i click outside of the element.

**Comment by Johnniey** [comment]
I think what @fajarlahhh's trying to say is that there's some weird behavior around the focus/opening of the dropdown, I noticed it aswell, at first i thought it was because the new jQuery update (3.4.0), but i'm starting to think its because of the Google Chrome 74 update.

**To recreate:**
Go to https://select2.org/i18n using Google Chrome 74 on Windows 10.
Click on any of the the demo select2 boxes, the boxes will open and stay open, it's almost impossible to close them.

Please let me know if you need more info or screenshots!

**Comment by Johnniey** [comment]
What I think @fajarlahhh is trying to say is that there's some weird behaviour around opening and closing the select2 boxxes since the 4.0.6 update. I noticed is myself aswell and after downgrading back to 4.0.5 it resolved the problem. Tested it in both Google Chrome and Firefox

**To recreate**

- Go to https://select2.org/i18n
- Click on any of the select2 boxxes.
- Try to close the box (Selecting an option, clicking outside the dropdown)

The dropdown won't close, it's almost impossible to close it.

If you need any further information, please let me know!

**Comment by bessone** [comment]
Same problem to me, reverting back to 4.0.6-rc1 the behaviour is back to normal

**Comment by ewgRa** [comment]
Same problem. Solved temporarily by reverting to 4.0.5

**Comment by afdev82** [comment]
I solved the problem updating to v4.0.7.rc0

**Comment by damienalexandre** [comment]
Fixed for me with 4.0.7-rc.0, looks like the 4.0.7 is tagged but not available on npm.
https://www.npmjs.com/package/select2

**Comment by 0lorak0** [comment]
I was on version 4.0.6 and had the issue where the dropdown would not close after selecting an option.
Updating to version 4.0.7 fixed the issue.

For reference, the code in 4.0.6 that was causing the issue was in the function
**Select2.prototype._registerEvents**.

Specifically, the event handler for "focus".  The code kept dropping into the else block, keeping the dropdown continuously open. Relevant code:

```javascript
Select2.prototype._registerEvents = function () {
    var self = this;
    
    this.on('focus', function () {
      self.$container.addClass('select2-container--focus');

      if (!self.$container.hasClass('select2-container--disabled') &&
          !self.isOpen()) {
        if (self.options.get('multiple')) {
          window.setTimeout(function () {
            self.open();
          },
          self.options.get('ajax') ? 300 : 100);
        }
        else {
          self.open();
        }
      }
    });
...
...
...
}
```

**Comment by kevin-brown** [comment]
This is fixed in 4.0.7, which was released a bit ago and is now available on all of the repositories (forgot NPM until now).
